content/de/about/we-are-using.md

```diff
@@ -280,6 +280,11 @@ permalink: '/about/we-are-using'
         </a>
     </li>
     <li>
+
+        <a href="//www.indev.net.br" target="_blank">
+            <img src="/assets/we-are-using/indevweb.png" alt="Indev Desenvolvimento web">
```

Comment:
I think adding a new company is not necessary to translate our docs. I'd rather have it in a separate PR.